modules/ROOT/partials/configuration/advtemplate_list.adoc

```diff
@@ -0,0 +1,44 @@
+[[advtemplate_list]]
+== `+advtemplate_list+`
+
+This options allows for the loading of templates from external data sources. 
```

Comment:
options -> option

Reply:
Urgent update, the `advtemplate_templates` options should not be mentioned in the public documentation. It is intended to use internally for testing or demo purposes. Once this option is set with a predefined constant template list, the plugin enables serverless mode, when all operations requests are redirected to the plugin's in-memory template database. It is important to understand that refreshing the page with the plugin in this mode resets the template list to the initial one. That is why using this option can mislead users and make them lose their data.

Reply:
To quote my commit message:

> 1. Removed `advtemplate_list.adoc ` from `/partials/configuration/`.
> 
> 2. Removed include statement pointing to `advtemplate_list.adoc ` from `advanced-templates.adoc`.
> 
> 3. Moved NOTE regarding unique id values requirement into `advanced-templates.adoc`.